src/DssSpell.sol

```diff
@@ -46,41 +96,259 @@ contract SpellAction {
+        // MedianAbstract(OsmAbstract(PIP_RENBTC).src()).kiss(PIP_RENBTC);
+        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_SPOT);
+        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_END);
+        // Set RENBTC Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_RENBTC = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(ILK_RENBTC_A, PIP_RENBTC);
+
+        // Set the RENBTC-A debt ceiling
+        VatAbstract(MCD_VAT).file(ILK_RENBTC_A, "line", 2 * MILLION * RAD);
+        // Set the RENBTC-A dust
+        VatAbstract(MCD_VAT).file(ILK_RENBTC_A, "dust", 500 * RAD);
+        // Set the Lot size
+        CatAbstract(MCD_CAT).file(ILK_RENBTC_A, "dunk", 50 * THOUSAND * RAD);
+        // Set the RENBTC-A liquidation penalty (e.g. 13% => X = 113)
+        CatAbstract(MCD_CAT).file(ILK_RENBTC_A, "chop", 113 * WAD / 100);
+        // Set the RENBTC-A stability fee (e.g. 1% = 1000000000315522921573372069)
+        JugAbstract(MCD_JUG).file(ILK_RENBTC_A, "duty", SIX_PERCENT_RATE);
+        // Set the RENBTC-A percentage between bids (e.g. 3% => X = 103)
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("beg", 103 * WAD / 100);
+        // Set the RENBTC-A time max time between bids
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("ttl", 6 hours);
+        // Set the RENBTC-A max auction duration to
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("tau", 6 hours);
+        // Set theRENBTC-A min collateralization ratio (e.g. 150% => X = 150)
+        SpotAbstract(MCD_SPOT).file(ILK_RENBTC_A, "mat", 175 * RAY / 100);
+
+        // Update RENBTC-A spot value in Vat
+        SpotAbstract(MCD_SPOT).poke(ILK_RENBTC_A);
+
+        // Add new ilk to the IlkRegistry
+        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_RENBTC_A);
+
+        // Update the changelog
+        CHANGELOG.setAddress("RENBTC", RENBTC);
+        CHANGELOG.setAddress("MCD_JOIN_RENBTC_A", MCD_JOIN_RENBTC_A);
+        CHANGELOG.setAddress("MCD_FLIP_RENBTC_A", MCD_FLIP_RENBTC_A);
+        CHANGELOG.setAddress("PIP_RENBTC", PIP_RENBTC);
+
+        //
+        // MIP27: Debt Ceiling Instant Access Module
+        //
+
+        // Give permissions to the MCD_IAM_AUTO_LINE to file() the vat
+        VatAbstract(MCD_VAT).rely(MCD_IAM_AUTO_LINE);
+
+        // Set ilks in MCD_IAM_AUTO_LINE
+        DssAutoLineAbstract(MCD_IAM_AUTO_LINE).setIlk(
+            "ETH-B", 50_000_000 * RAD, 5_000_000 * RAD, 12 hours
```

Comment:
We should do `50 * MILLION * RAD` here for consistency with the setters above.

Reply:
and `5 * MILLION * RAD` respectively

Reply:
Yeah I'm not sure if we should do that or move everything to `_`. The global debt ceiling is much easier to be set this way than using the units.

Reply:
I agree with this, and only use this format for more complicated values (like the `Line` in this one)

Reply:
ok good, changing it!

Reply:
Done!